pkg/larker/fs/github/github.go

```diff
@@ -36,18 +39,32 @@ func New(owner, repo, reference, token string) (*GitHub, error) {
 	if err != nil {
 		return nil, err
 	}
+	fileInfosCache, err := lru.New(16)
```

Comment:
Can probably have a bigger cache just for the informations without contents. 🤔

Reply:
Indeed, see https://github.com/cirruslabs/cirrus-cli/pull/474/commits/f343965db8ac5f10976258cf8b64fa8c7ddd611b.